### ISSUE TITLE
Fortran documentation mentions as_string

### ISSUE THREAD

**Issue by LourensVeen** [multiscale/muscle3#228]
https://muscle3.readthedocs.io/en/latest/fortran.html#describing-fortran-implementations

In the reference box, it says
```
    if (LIBMUSCLE_DataConstRef_is_a_character(d9j))
        s = LIBMUSCLE_DataConstRef_as_string(d9j)
```
which should be `LIBMUSCLE_DataConstRef_as_character` of course.

Thanks to Severin for reporting!

**Comment by LourensVeen** [comment]
`LIBMUSCLE_as_int8` is also wrong, and should be fixed.

**Comment by LourensVeen** [comment]
Released with 0.7.0.
